src/index.js

```diff
@@ -1,23 +1,27 @@
 import './style.scss';
-import data from './modules/Data.js';
+import { addScoreToApi, scoreList } from './modules/api.js';
+import List from './modules/List.js';
 
-const refresh = document.getElementById('referesh');
-refresh.addEventListener('click', () => {
+const referesh = document.getElementById('referesh');
+const username = document.getElementById('name');
+const score = document.getElementById('score');
+const add = document.getElementById('btn');
+const listItem = document.getElementsByClassName('item');
+
+referesh.addEventListener('click', () => {
   window.location.reload();
 });
 
-const ScoreContainer = document.querySelector('.Scores');
+if (!listItem.length) {
+  scoreList();
+}
 
-data.forEach((index) => {
-  const List = document.createElement('tr');
-  List.className = 'listScore';
-  ScoreContainer.appendChild(List);
-  const item = document.createElement('td');
-  item.className = 'itemScore';
-  item.textContent = index.name;
-  List.appendChild(item);
-  const item1 = document.createElement('td');
-  item1.className = 'itemScore';
-  item1.textContent = index.score;
-  List.appendChild(item1);
-});
\ No newline at end of file
+add.addEventListener('click', () => {
+  const newItemIndex = listItem.length;
+  if (username.value && score.value && score.value <= 9999999) {
+    addScoreToApi(username.value, score.value);
+    List(username.value, score.value, newItemIndex);
```

Comment:
- [ ] Based on the requirements of this project, the score list should only change when the refresh button is clicked and not immediately when the submit button is clicked. You may raise this line of code when you called the click event listener function on the refresh button.




![](https://media.giphy.com/media/JrGUGeWPlBuOe0GoQr/giphy.gif)

Reply:
> Thank you for the review 
> best